arrow-core/src/main/kotlin/arrow/core/extensions/const/divide/ConstDivideInstance.kt

```diff
@@ -54,17 +50,22 @@ fun <O, A, B, Z> divide(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "product(MO, arg1)",
-    "arrow.core.product"
+    "mapN(MO, arg0, arg1, arg2).map(arg3)",
+    "arrow.core.mapN"
   ),
   DeprecationLevel.WARNING
 )
-fun <O, A, B> Kind<Kind<ForConst, O>, A>.product(MO: Monoid<O>, arg1: Kind<Kind<ForConst, O>, B>):
-  Const<O, Tuple2<A, B>> = arrow.core.Const.divide<O>(MO).run {
-  this@product.product<A, B>(arg1) as arrow.core.Const<O, arrow.core.Tuple2<A, B>>
-}
+fun <O, A, B, C, Z> divide(
+  MO: Monoid<O>,
+  arg0: Kind<Kind<ForConst, O>, A>,
+  arg1: Kind<Kind<ForConst, O>, B>,
+  arg2: Kind<Kind<ForConst, O>, C>,
+  arg3: Function1<Z, Tuple3<A, B, C>>
+): Const<O, Z> = arrow.core.Const
+  .divide<O>(MO)
+  .divide<A, B, C, Z>(arg0, arg1, arg2, arg3) as arrow.core.Const<O, Z>
 
-@JvmName("product1")
+@JvmName("divide")
```

Comment:
We can't change these names. This is a binary breaking change, but these names don't really matter since they will be removed in `0.13.0`.

Reply:
Oh I'm not changing the name.. only grouped the functions together.. divide with divide's and product with product's

Reply:
Oh, you changed the order of the generated functions in the file 😁 Missed that.